Metron/.github/ISSUE_TEMPLATE/bug.md

```
@@ -0,0 +1,25 @@
+<!-- Clear, concise description -->
+
+## Reproduction steps
+1.
+2.
+3.
+
+## Expected behavior
+
+## Actual behavior
+
+## Environment
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)